packages/core/bootstrap/test/unit/cache-key.test.ts

```diff
@@ -11,12 +12,11 @@ describe('Cache key middleware', () => {
         from: 'btc',
         to: 'eth',
         overrides: {
-          btc: 'bitcoin',
+          btc: { bitcoin: 'bitcoin' },
```

Comment:
While the way you have changed it is still an accurate test, let's structure these how they should actually look

```suggestion
          adapterName: { btc: 'bitcoin' },
```